server/src/main/kotlin/delta/codecharacter/server/user/public_user/PublicUserEntity.kt

```diff
@@ -22,5 +22,5 @@ data class PublicUserEntity(
     val ties: Int,
     val isActivated: Boolean = true,
     val score: Double,
-    val isDailyChallengeCompleted: Boolean = false,
+    val dailyChallengeHistory: HashMap<UUID, DailyChallengeHistory>
```

Comment:
It should be a `HashMap` of  Day number and a `Dataclass` of *Score*,*DailychallengeEntity*